src/python/pants/bin/local_pants_runner.py

```diff
@@ -86,7 +87,17 @@ def _init_graph_session(
 
         global_scope = options.for_global_scope()
 
-        dynamic_ui = global_scope.dynamic_ui
+        if global_scope.v2:
+            dynamic_ui = resolve_conflicting_options(
+                old_option="v2_ui",
+                new_option="dynamic_ui",
+                old_scope="",
```

Comment:
Probably better to use the name `GLOBAL_SCOPE` from `pants.option.scope`, both here and the above file.

Reply:
Would agree on using the constant at this callsite, but at the other one it would introduce a dependency on a package that we don't want... could maybe leave a comment about that though.

Reply:
Ah, true. Sounds good.